### PR TITLE
22 favourites details modal

### PR DESCRIPTION
**Pull request by JustinDotMorris**

### CODE REVIEW

File: src/components/BusinessCard/BusinessCard.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@use "../../styles/variables" as *;
+
+.button--Details{
+    color: $white;
+    background-color: $green-color-palette;
+    border-radius: 2rem;
+}
```
Comment (DanForder) [inline src/components/BusinessCard/BusinessCard.scss:7]: this should be added to button.scss
and classname should follow BEM convention: `.button--details`

File: src/components/BusinessCard/BusinessCard.test.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  expect(container).toMatchSnapshot();
+});
+
+it("should check if the click of the details button shows the details pane"),
+  () => {
+    const { container } = customRender(<BusinessCard />);
+    const button = screen.getByRole("button");
+
+    userEvent.click(button);
+    expect(container);
```
Comment (DanForder) [inline src/components/BusinessCard/BusinessCard.test.jsx:17]: what do you expect? do we need to check that there the modal appears?

File: src/components/DetailsPane/DetailsPane.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <p>{card.description}</p>
+        <p>{card.weekdayOpening}</p>
+        <p>{card.weekendOpening}</p>
+        <p>{card.address}</p>
+        <p>{card.price}</p>
+        <p>{card.image}</p>
+        <div className="detailsPane__SaveBtnContainer">
+          <Button buttonText="Save" buttonName="Save" />
+        </div>
+      </div>
```
Comment (DanForder) [inline src/components/DetailsPane/DetailsPane.jsx:21]: BEM classes should be lowercase hyphenated: `details-pane`, `details-pane__container` etc.

File: src/components/DetailsPane/DetailsPane.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    width: 3rem;
+}
+.detailsPane__SaveBtnContainer{
+    display: flex;
+    justify-content: center;
+}
+.button--Save{
+    background-color: $orange-color-palette;
+    border-radius: 1.5rem;
+}
```
Comment (DanForder) [inline src/components/DetailsPane/DetailsPane.scss:38]: this class should be on button.scss and lowercase hypenated too: `.button--save`

**Review by DanForder — CHANGES_REQUESTED** [review]

**Review by GeorgeSwift16 — CHANGES_REQUESTED** [review]
Cracking work all functions and looks good! Can we copy and paste the details button to create a remove button next to it. just needs the background going red and text changing to remove, the rest looks great. - ignore the positioning /aligning of the button as we have a user story to do the card design. just get it in there so its within the card component for someone to do the styling. Cheers! We also have a couple of errors relating to unused variables in two of the test files so can the import statement for these vars get deleted. https://gyazo.com/553149789b9baebe338eb808d1620e40

**Review by DanForder — CHANGES_REQUESTED** [review]
code all good, just need to get the actions passing (got a few linting errors)

**Review by GeorgeSwift16 — APPROVED** [review]
Meets all trello criteria and has latest changes from main for favicon merged

**Review by DanForder — CHANGES_REQUESTED** [review]
still failing the linting stage, please address these and we can get this merged

**Review by DanForder — APPROVED** [review]

**Pull request merged**